geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommandDUnitTest.java

```diff
@@ -385,6 +387,79 @@ public void createMappingPartitionedUpdatesServiceAndClusterConfigForServerGroup
     });
   }
 
+  // TODO: need to test this use case
```

Comment:
What is the point of having all this test code? Is it currently never called? I don't understand this TODO comment